### ISSUE TITLE
uninitialized constant Falcon::Command::Top::Logger

### ISSUE THREAD

**Issue by mpugach** [socketry/falcon#44]
hi

Gemfile.lock
```
GEM
  remote: https://rubygems.org/
  specs:
    async (1.13.0)
      nio4r (~> 2.3)
      timers (~> 4.1)
    async-container (0.8.1)
      async (~> 1.0)
      async-io (~> 1.4)
    async-http (0.37.7)
      async (~> 1.6)
      async-io (~> 1.16)
      http-protocol (~> 0.10)
    async-io (1.17.2)
      async (~> 1.3)
    falcon (0.19.6)
      async-container (~> 0.8.0)
      async-http (~> 0.37.2)
      async-io (~> 1.9)
      http-protocol (~> 0.10.0)
      localhost (~> 1.1)
      rack (>= 1.0)
      samovar (~> 1.3)
    http-hpack (0.1.1)
    http-protocol (0.10.1)
      http-hpack (~> 0.1.0)
    localhost (1.1.4)
    mapping (1.1.1)
    mustermann (1.0.3)
    nio4r (2.3.1)
    rack (2.0.6)
    rack-protection (2.0.5)
      rack
    rainbow (3.0.0)
    samovar (1.9.1)
      mapping (~> 1.0)
      rainbow (>= 2.0, < 4.0)
    sinatra (2.0.5)
      mustermann (~> 1.0)
      rack (~> 2.0)
      rack-protection (= 2.0.5)
      tilt (~> 2.0)
    tilt (2.0.9)
    timers (4.2.0)

PLATFORMS
  ruby

DEPENDENCIES
  falcon
  sinatra

RUBY VERSION
   ruby 2.5.3p105

BUNDLED WITH
   1.17.2
```

config.ru
```
require 'sinatra/base'

class MyApp < Sinatra::Base
  get '/ololo' do
    'up'
  end
end

use MyApp
run lambda {|env| [404, {}, []]}
```

command
```
falcon serve
```

what do I do wrong?

**Comment by mpugach** [comment]
ok, seems locking `async` to `1.12.0` does the job

**Comment by ioquatix** [comment]
Sorry I am about to release a small fix for this.

**Comment by ioquatix** [comment]
`lib/async/logger.rb` was including `logger` but it's no longer used. Upstream code depended on this implicit require, so I've added it back until upstream is fixed.

**Comment by ioquatix** [comment]
Fixed in async v1.13.1

**Comment by mpugach** [comment]
thank you :)
